utils/aeternity.js

```diff
@@ -182,7 +184,9 @@ class Aeternity {
 
   getTokenMetaInfoCacheAccounts = async (address) => {
     const fetchData = async () => {
-      const contract = await this.client.getContractInstance(TOKEN_CONTRACT_INTERFACE, {contractAddress: address});
+      const contract =  this.tokenContracts[address]
```

Comment:
shouldn't you add it to the object in the else-case? or where is `this.tokenContracts` populated?

Reply:
you are right, fixing it